modules/virt-creating-configmap.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+// * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
+[id='virt-creating-configmap_{context}']
+= Creating a custom operating system ConfigMap
+
+If the operating system of the Red Hat Virtualization virtual machine does not exist in the default ConfigMap of the `vm-import-controller` Custom Resource (CR), you can create a custom ConfigMap to map the operating system to a common template. The template is used to create the new virtual machine in {VirtProductName}.
+
+.Procedure
+
+. Create a file named `configmap.txt` that maps the RHV virtual machine's operating system to a common template, as in the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: os_configmap
```

Comment:
Please replace `os_configmap` with `os-configmap`
(configmap names should follow DNS-1123 format which forbids underscore)

Reply:
Done